crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -338,6 +338,61 @@ pub struct StandardMaterial {
     #[dependency]
     pub occlusion_texture: Option<Handle<Image>>,
 
+    /// An extra thin translucent layer on top of the main PBR layer. According
+    /// to the Filament spec, this is commonly used for materials such as car
```

Comment:
I would remove the mention of filament.

Reply:
Yeah, I just mentioned Filament there because I lifted the line from its documentation and was worried about plagiarism. I switched it to just "This is typically used for painted surfaces."